src/GitVersionTfsTask/task.json

```diff
@@ -35,6 +35,15 @@
       "helpMarkDown": "Whether to update versions in the AssemblyInfo files"
     },
     {
+      "name": "updateAssemblyInfoFilename",
+      "type": "string",
+      "label": "Update Assembly File",
+      "defaultValue": "",
+      "required": false,
+      "helpMarkDown": "Update versions in specified file",
```

Comment:
Doesn't look like: https://github.com/GitTools/GitVersion/blob/039a43b092134c6a42902fc162fcb4cbc1265c14/src/GitVersionExe/AssemblyInfoFileUpdate.cs#L52. Wildcard support is something that needs first be implemented in GitVersion. But mentioning that only full file names are supported is still useful (since most of the OOTB VSO task support patterns or minimatch).